apps/frontend/src/app/PageScanner/index.tsx

```diff
@@ -31,7 +32,16 @@ import AdScanner from './AdeptiScanner.png'
 import Amenoma from './Amenoma.png'
 import Artiscan from './artiscan.png'
 import GIScanner from './GIScanner.png'
-
+import AkashaScanner from './AkashaScanner.png'
+import SvgIcon from '@mui/material/SvgIcon'
+const Weapons = () => (
```

Comment:
Should be able to reuse `Assets.svg.anvil` the same way it is used in `Header.tsx`

Reply:
i mean i tried but i couldn't. do you mean like `import Assets.svg.anvil from ____` and then `<SqBadge ____ > <Anvil />`?

Reply:
You'll have to `import Assets from '../Assets/Assets'` then do it as `<SqBadge>{Assets.svg.anvil}</SqBadge>`